pkg/handlers/app.go

```diff
@@ -55,6 +55,20 @@ type SendCustomAppMetricsRequest struct {
 
 type CustomAppMetricsData map[string]interface{}
 
+type SendAppInstanceTagsRequest struct {
+	Data AppInstanceTagsData `json:"data"`
+}
+
+type AppInstanceTagsData map[string]interface{}
+
+func (a AppInstanceTagsData) ToMapString() map[string]string {
+	m := make(map[string]string)
+	for k, v := range a {
+		m[k] = v.(string)
```

Comment:
this will panic if it's not a string

Reply:
Yeah, this is only called after the validation check. I can remove it make this more explicit in the handler.